scripts/build/generateDocs.mjs

```diff
@@ -105,8 +105,16 @@ export function generateReadmeUrl(tag = 'master', variantOverride = '') {
   }
 
   const baseRepoUrl = 'https://raw.githubusercontent.com/AlaskaAirlines/WC-Generator'
-  if (tag !== 'master') {
-    return `${baseRepoUrl}/refs/tags/${tag}/componentDocs/README` + variantString + '.md';
+
+  // check if tag starts with 'vX' since our tags are `v4.0.0`
+  const isTag = /^v\d.*$/.test(branchOrTag);
```

Comment:
this is a great catch! I wanted to leave open the `startsWith('v')` as our branch names are `<first-initial><lastname>/<feature>` in case someone has a `v` name but your combination regex + startsWith handles both